### PR TITLE
Access control documentation

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Control Access in the Choreo Console
+
+In the Choreo console, you have the ability to manage access to projects and the actions that can be performed within them. Administrators have the capability to restrict project access to specific user groups. This feature is useful when you need certain user groups to have access to particular projects or for a set of projects.
```
Comment (Nashaath) [inline en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:3]: Need to change "console" to "Console".

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Choreo uses **Roles**, **Groups**, and a **Mapping level** to control access to the Choreo console as follows:
```
Comment (Nashaath) [inline en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:5]: Need to change "console" to "Console".

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    that users in the group inherit the permissions granted by that role only within the context of the specified project.
+                    For example, If a user has edit_project permission, that user can only edit the specified project.
+
+
+In Choreo, authorization operates by assigning a role to a group at a specified level. The level at which the role is assigned determines the extent of permissions granted to users.
+
+!!! info
+    In Choreo, organization-level permissions take precedence over project-level permissions.
+
+To elaborate further, refer to the following image.
```
Comment (Nashaath) [inline en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:24]: Shall we use the term diagram here?

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+In Choreo, authorization operates by assigning a role to a group at a specified level. The level at which the role is assigned determines the extent of permissions granted to users.
+
+!!! info
+    In Choreo, organization-level permissions take precedence over project-level permissions.
+
+To elaborate further, refer to the following image. 
+
+The following image depicts a role-group assignment at a specific resource level. In the diagram, an admin user has assigned the Developer role to all members of the Engineering group within the Engineering Project. This grants users in the Engineering group the ability to perform all actions allowed by the Developer role within the Engineering Project.
```
Comment (Nashaath) [inline en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:26]: We've used the term diagram in the next sentence. shall we use the same here too?

File: en/docs/administer/inviting-members.md [hunk 5]
```diff
@@ -1,15 +1,3 @@
...
-- **Admin**: An administrator is responsible for all administration tasks, including user management, customizing the Developer Portal, managing projects, enabling analytics, managing domains, etc.
-- **Billing Admin**: Is responsible for billing administration that includes viewing tiers, creating and viewing organizations, managing invoices, viewing and creating subscriptions, and viewing and creating payment methods.
-- **Choreo DevOps**: A Choreo DevOps user is a user with access to the Choreo DevOps portal, enabling them to actively manage, ensure dependable deployment, and monitor components.  
-- **External API subscriber**: External API subscribers are API consumers who have access only to the API Developer Portal. They can join an organization with the sole purpose of consuming APIs.
-- **Environment Manager (Deprecated)**: Manages deployment environments.
+An organization administrator can invite members to the organization by assigning them specific groups. Invited members receive an invitation via email. An invited member must accept the invitation to join the organization and access the resources of that organization.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/administer/inviting-members.md:3]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Choreo uses **Roles**, **Groups**, and a **Mapping level** to control access to the Choreo console as follows: 
+
+- **Role** : Role is a collection of permissions. Choreo has a [predefined set of roles](groups-and-roles.md#roles) with permissions assigned to them. [Learn more](groups-and-roles.md#roles)
+- **Group** : Group is a collection of users. A user group requires a role or multiple roles to be assigned to it so that the users in those groups get the relevant permissions via the assigned roles. [Learn more](groups-and-roles.md#groups)
+
+- **Mapping level** : A mapping level defines the extent at which a role-group mapping can be done. Choreo has two defined resource levels.
+    - **Organization** : You can assign a role to a group or associate a group with a role within the organization. This ensures that    
+                         all users in a group inherit the permissions granted by that role across all organizational resources.
+                         For example, if a user has edit_project permission, that user can edit all the projects in the organization.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:13]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    - **Project** : You can assign a role to a group or associate a group with a role within a specific project resource. This ensures 
+                    that users in the group inherit the permissions granted by that role only within the context of the specified project.
+                    For example, If a user has edit_project permission, that user can only edit the specified project.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:16]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md (hunk 1 above)
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:3]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md (hunk 2 above)
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:5]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md (hunk 3 above)
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:24]: done

File: en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md (hunk 4 above)
Comment (shaniR) [reply en/docs/authentication-and-authorization/access-control/control-access-in-the-choreo-console.md:26]: done

**Review by vlgunarathne — APPROVED** [review]

**Review by Nashaath — APPROVED** [review]

**Pull request merged**